exporter/containerimage/writer.go

```diff
@@ -421,6 +422,12 @@ func (ic *ImageWriter) rewriteRemoteWithEpoch(ctx context.Context, opts *ImageCo
 		i, desc := i, desc
 		info, err := cs.Info(ctx, desc.Digest)
 		if err != nil {
+			if errors.Is(err, errdefs.ErrNotFound) {
+				// blobs for base image layers can be missing here
```

Comment:
Isn't it possible to detect that this is base image layer before and avoid calling to `Info()` then.

Reply:
I don’t think so, as we have to compute diff ID, and it is not always present in desc.Annotations

Reply:
But then how do we know that this is expected and we are not just creating a broken image with missing blobs.

Reply:
Updated the PR to check `desc.Annotations` rather than `info.Labels`.

Also updated the test to verify that all the layer blobs are present in the output image.